target/linux/ramips/dts/mt7620a_youku.dtsi

```diff
@@ -88,10 +85,9 @@
 				read-only;
 			};
 
-			partition@50000 {
+			firmware: partition@50000 {
 				compatible = "denx,uimage";
 				label = "firmware";
-				reg = <0x50000 0x1fb0000>;
```

Comment:
Please leave a comment like the following instead of the reg property here:
`/* reg property is set based on flash size in DTS files */`

Reply:
Now added, thx.